ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```
@@ -98,6 +114,10 @@ wan_router:
               cv_pathfinder_internet_exit:
                 policies:
                   - name: DIRECT-EXIT-POLICY-1
+              bgp:
+                peer_as: 1234
+                prefix_list_in: "allow_default"
+                prefix_list_out: "allow_default"
```

Please use upper case so it is easier to tell apart in the output config.

```suggestion
                prefix_list_in: ALLOW-DEFAULT
                prefix_list_out: ALLOW-DEFAULT
```

---

Also please make the key resemble the global key, so `ipv4_prefix_list_in/out`